Though anaconda is booted with option `--method=livecd`, it does NOT boot a
real live cd environment. Set it with False for oe.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

---
 pyanaconda/core/configuration/system.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyanaconda/core/configuration/system.py b/pyanaconda/core/configuration/system.py
index 3e539dbed1..1716ea1a0e 100644
--- a/pyanaconda/core/configuration/system.py
+++ b/pyanaconda/core/configuration/system.py
@@ -48,7 +48,7 @@ class SystemSection(Section):
     @property
     def _is_live_os(self):
         """Are we running in the live OS?"""
-        return self._type is SystemType.LIVE_OS
+        return False
 
     @property
     def _is_booted_os(self):
